metricbeat/docs/fields.asciidoc

```diff
@@ -1083,6 +1083,13 @@ type: integer
 The amount of CPU time spent on low-priority processes.
 
 
+==== system.core.nice_p
```

Comment:
Could we call it in metricbeat nice_pct: https://github.com/elastic/beats/blob/master/metricbeat/module/doc.go#L34 p seems for me too generic. But this is no blocker for this PR.